src/details_window.cr

```diff
@@ -50,13 +50,13 @@ class DetailsWindow < Window
       @card.activities.map { |activity| CardAction.new(activity) }.each do |activity|
         pad.attron(NCurses::Attribute::BOLD | NCurses::Attribute::UNDERLINE)
         wrap(activity.title, @width - 2).each_line do |line|
-          pad.addstr(line)
-          pad.addstr("\n") unless line.ends_with?("\n")
+          pad.addstr(line.rstrip)
+          pad.addstr("\n") unless line.size == @width - 2
```

Comment:
I don't love this. I feel there's a more elegant solution. I wrote a better explanation of why it's needed in commit e934a26fef1a412dab438120567f2fb49e5f4b7f.